### ISSUE TITLE
why BookieNettyServer’s parentGroup and childGroup is same EventLoopGroup?

### ISSUE THREAD

**Issue by luoxn28** [apache/bookkeeper#2104]
**QUESTION**
as code, If use independent EventLoopGroup, connect event and write/read event  don't interact other. so, why  BookieNettyServer’s parentGroup and childGroup is same EventLoopGroup?
```java
private void listenOn(InetSocketAddress address, BookieSocketAddress bookieAddress) throws InterruptedException {
        if (!conf.isDisableServerSocketBind()) {
            ServerBootstrap bootstrap = new ServerBootstrap();
            bootstrap.option(ChannelOption.ALLOCATOR, allocator);
            bootstrap.childOption(ChannelOption.ALLOCATOR, allocator);
            bootstrap.group(eventLoopGroup, eventLoopGroup); // <-----
// ...
```
Looking forward to your reply, thans.

**Comment by sijie** [comment]
@luoxn28 since the bookie is implemented in asynchronous way, I don't think it really matter if using two separate EventLoopGroup.

**Comment by luoxn28** [comment]
@sijie You're right, using an EventGroupLoop doesn't make much of a difference, especially in an asynchronous way, which makes little difference to the client.But it's better to use a different EventGroupLoop for isolation.

**Comment by sijie** [comment]
@luoxn28 agreed. Are you interested in contributing a feature to add an option to make it able to configure using a different EventGroupLoop for isolation?

**Comment by shoothzj** [comment]
closed by #3153
